src/rules/no-native-attributes.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @fileoverview Disallows use of native attributes as properties
+ * @author Pascal Schilp <https://github.com/thepassle>
+ */
+
+import { Rule } from 'eslint';
+import * as ESTree from 'estree';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      description: 'Disallows use of native attributes as properties',
+      recommended: false,
+      url: 'https://github.com/43081j/eslint-plugin-lit/blob/master/docs/rules/no-native-attributes.md'
+    },
+    schema: [],
+    messages: {
+      noNativeAttributes:
+        'Avoid using global native attributes as properties, ' +
+        'this can have unintended effects'
+    }
+  },
+
+  create(context): Rule.RuleListener {
+    // variables should be defined here
+
+    // Taken from https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes
+    const NATIVE_ATTRS = [
```

Comment:
maybe can put this outside the rule definition (top of this file) since its not necessarily a context-specific variable